### PR TITLE
Statement to clarify where the tracer resolver should be used

### PR DESCRIPTION
**Pull request by objectiser**
As discussed in #15, a paragraph to clarify where the TracerResolver should be used.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,12 @@
 # java-tracerresolver
 Resolver API for OpenTracing Tracer implementations.
 
+NOTE: The Tracer Resolver mechanism is only intended to be used at application startup/initialization time. This responsibility
+should be handled by the application, possibly using some runtime specific support (e.g. providing a
+Tracer @Bean in Spring Boot, or a CDI producer). Framework integrations used to instrument specific technologies used
+by an application should not use this library.
+
```
Comment (sjoerdtalsma) [inline README.md]: Nit: I'd start sentences on a new line for source readability (they'll be combined on-screen anyway) e.g.:
>NOTE: The Tracer Resolver mechanism is only intended to be used at application startup/initialization time.
This responsibility should be handled by the application, possibly using some runtime specific support
(e.g. providing a Tracer @Bean in Spring Boot, or a CDI producer). 
Framework integrations used to instrument specific technologies used by an application should not use this library.

Comment (sjoerdtalsma) [reply README.md]: Nit: add backticks around `@Bean` otherwise we _github mention_ mr Bean :wink:

Comment (sjoerdtalsma) [reply README.md]: Nit: I would make the following bold for added emphasis:
**NOTE:** and 
..should **not** use this library..

**Review by sjoerdtalsma — APPROVED** [review]
Aside from a few nits, I agree with the content.

**Comment by sjoerdtalsma** [comment]
@pavolloffay Does this resolve the issue for you?

File: README.md [hunk 2]
```diff
@@ -4,10 +4,10 @@
...
 Resolver API for OpenTracing Tracer implementations.
 
-NOTE: The Tracer Resolver mechanism is only intended to be used at application startup/initialization time. This responsibility
-should be handled by the application, possibly using some runtime specific support (e.g. providing a
-Tracer @Bean in Spring Boot, or a CDI producer). Framework integrations used to instrument specific technologies used
-by an application should not use this library.
+**NOTE:** The Tracer Resolver mechanism is only intended to be used at application startup/initialization time.
+This responsibility should be handled by the application, possibly using some runtime specific support (e.g. providing a
+Tracer `@Bean` in Spring Boot, or a CDI producer).
+Framework integrations used to instrument specific technologies used by an application should **not** use this library.
```
Comment (sjoerdtalsma) [inline README.md]: maybe add what they _should_ use instead?
> Framework integrations used to instrument specific technologies used by an application should **not** use this library.

-->
> Framework integrations used to instrument specific technologies should **not** use this library, but should allow a Tracer to be injected instead, with fallback to the GlobalTracer.

**Review by sjoerdtalsma — COMMENTED** [review]
One more comment.

**Comment by pavolloffay** [comment]
It looks great :+1:

**Pull request merged**